jsk_robot_common/speak_and_wait_recovery/test/test_speak_and_wait.py

```diff
@@ -24,14 +26,13 @@ def test_speak_and_wait_recovery(self):
         self.msg = None
         self.action_server = actionlib.SimpleActionServer( '/sound_play', SoundRequestAction, execute_cb=self.handler, auto_start=False)
         self.action_server.start()
+        msg = SoundRequest()
         while not rospy.is_shutdown():
             if self.msg is not None:
                 msg = self.msg
                 break
 
-        self.assertEqual(msg.sound, -3)
-        self.assertEqual(msg.command, 1)
-        self.assertEqual(msg.volume, 1.0)
```

Comment:
just out of curiosity, why this fixes test? If `speak_and_wait_recovery_simple_node` receves `msg.arg == "test"` then it also contaions, `msg.sound`, `msg.command` and `msg.volume`.
```
        name="speak_and_wait_recovery_simple_node"
        >
        <rosparam>
          recovery:
            speak_text: 'test'
```

Reply:
I think it will not fix test. This comes from my local changes for output simplicity.

Reply:
Sorry, this is trial for fixing. Some changes may means nothing.

Reply:
I see, I  thought you solved this problem, because it passed test -> https://github.com/jsk-ros-pkg/jsk_robot/actions/runs/2205653284/attempts/2